src/server/lib/ab.ts

```diff
@@ -60,9 +72,82 @@ export const selectVariant = <V extends Variant, T extends Test<V>>(test: T, mvt
     return selectWithSeed(mvtId, seed, test.variants);
 };
 
-export const selectVariantNonSticky = <V extends Variant, T extends Test<V>>(test: T): V => {
-    const index = Math.floor(Math.random() * test.variants.length);
-    return test.variants[index];
+const selectVariantWithMethodology = <V extends Variant, T extends Test<V>>(
+    test: T,
+    mvtId: number,
+    banditData: BanditData[],
+    methodology: Methodology,
+): V | undefined => {
+    if (methodology.name === 'EpsilonGreedyBandit') {
+        return selectVariantUsingEpsilonGreedy(banditData, test, methodology.epsilon);
+    }
+    return selectVariantUsingMVT<V, T>(test, mvtId);
+};
+
+const addMethodologyToTestName = (testName: string, methodology: Methodology): string => {
+    if (methodology.name === 'EpsilonGreedyBandit') {
+        return `${testName}_EpsilonGreedyBandit-${methodology.epsilon}`;
```

Comment:
I just realised a flaw in cases where this is used (i.e. where we have more than 1 methodology).
In `banditData.ts` we don't currently append the methodology to the test name when looking up data for a given test.
This isn't currently an issue because nothing is using bandit tests yet